packaging/CMakeLists.txt

```diff
@@ -2,19 +2,21 @@
 # OS-independent CPack configuration.
 set(CPACK_COMPONENTS_ALL "")
 set(CPACK_OUTPUT_FILE_PREFIX "dist")
+set(CPACK_PACKAGE_NAME "ZeekAgent")
 set(CPACK_PACKAGE_CONTACT "info@zeek.org")
 set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Zeek Agent is an endpoint monitoring tool reporting host information to Zeek.")
 set(CPACK_PACKAGE_HOMEPAGE_URL "https://zeek.org")
 set(CPACK_PACKAGE_RELOCATABLE yes)
 set(CPACK_PACKAGE_VENDOR "Zeek Project")
-set(CPACK_PACKAGE_VERSION "${ZEEK_AGENT_VERSION}")
+set(ZEEK_BINARY_VERSION_NAME "zeek-agent-${ZEEK_AGENT_VERSION}")
```

Comment:
How about calling this `PACKAGE_FILE_NAME_BASE`? I believe that's what it is. And the move it out of this `CPACK_*` block to the beginning.